Add gitfetch (#1213)
aharris88/awesome-cli: readme.md

- [cli-github](https://github.com/IonicaBizau/cli-github) - Fancy GitHub client.
- [hub](https://github.com/github/hub) - Make git easier to use with GitHub.
- [git-labelmaker](https://github.com/himynameisdave/git-labelmaker) - Edit GitHub labels.

> - [gitfetch](https://github.com/Matars/gitfetch) - Neofetch-style stats for your git profiles.

Reviewer: ```suggestion
- [gitfetch](https://github.com/Matars/gitfetch) - Neofetch-style stats of your git forge.
```
